rasgotransforms/rasgotransforms/transforms/clean/clean.sql

```diff
@@ -11,7 +11,7 @@
 {%- macro get_select_column(name, col) -%}
     {%- set source_col = 'cast(' + name + ' as ' + col.type + ')'  if col.type is defined else name-%}
     {% set output_col_name = name if col.name is not defined else cleanse_name(col.name) %}
-    {%- set impute_expression = '' -%}
+    {%- set impute_expression = none -%}
```

Comment:
(asking to learn, not bc i know anything) jinja wants none instead of None?

Reply:
No actually this needs to be NULL since it's going straight into our SQL string